src/components/common/ProductBox/ProductBoxContainer.js

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux';
+import ProductBox from './ProductBox';
+import { setCompare, getCount } from '../../../redux/compareRedux';
+
+const mapStateToProps = state => ({
+  count: getCount(state),
```

Comment:
It would be good to make this prop name more precise. `count` says that it is the number but doesn't really explain what it really is. Is it the number of products? Or something else? I think `compareCount` would be more meaningful name.